www/docs/en/dev/config_ref/index.md

```diff
@@ -234,6 +234,23 @@ platform. See [Customize icons topic](images.html) for more information.
    <plugin name="cordova-plugin-device" spec="https://github.com/apache/cordova-plugin-device.git#1.0.0" />
    ```
 
+### variable
+   Persists the value of a CLI variable to be used when restoring a plugin during a prepare. See the [preference element][plugin_preference]
```

Comment:
It will be great to specify that edits to the value in config.xml do not take effect without removing and adding the plugin. This is added by --save option in the CLI.